janitor/functions.py

```diff
@@ -2201,11 +2201,50 @@ def transform_column(
     column_name: Hashable,
     function: Callable,
     dest_column_name: str = None,
+    elementwise: bool = True,
 ) -> pd.DataFrame:
     """
     Transform the given column in-place using the provided function.
 
-    This method mutates the original DataFrame.
+    Functions can be applied one of two ways:
+
+    - Element-wise (default; ``elementwise=True``)
+    - Column-wise  (alternative; ``elementwise=False``)
+
+    If the function is applied "elementwise",
+    then the first argument of the function signature
+    should be the individual element of each function.
+    This is the default behaviour of ``transform_column``,
+    because it is easy to understand.
+    For example:
+
+    .. code-block:: python
+
+        def elemwise_func(x):
+            modified_x = ... # do stuff here
+            return modified_x
+
+        df.transform_column(column_name="my_column", function=elementwise_func)
+
+    On the other hand, columnwise application of a function
+    behaves as if the function takes in a pandas Series
+    and emits back a sequence that is of identical length to the original.
+    One place where this is desirable
+    is to gain access to `pandas` native string methods,
```

Comment:
Should this be \``pandas\`` or did you want this to be italics?

I believe there other cases in the project where \` is used instead of \`` in docstrings due to slips between .md and .rst. But it's possible the author _did_ intend to generate italic text.

Reply:
Ahhhh, I kinda sorta want to move over the docs to mkdocs to take advantage of Markdown docstrings, but I'm hesitant to undo so many contributors' work making the docs work with Sphinx.

Reply:
In any case, good catch, I'll change it to double backticks.

Reply:
I'm not volunteering right now, but supposedly it could be converted [via pandoc](https://news.ycombinator.com/item?id=17717513). 

Would you be looking for a conversion, or complete re-write?